RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -244,15 +247,19 @@
 fastSim.toModify(highPtTripletStepTracks,TTRHBuilder = 'WithoutRefit')
 
 # Final selection
+from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
+highPtTripletStep = TrackMVAClassifierPrompt.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorHighPtTripletStep_Phase1'),
+     src = 'highPtTripletStepTracks',
+     qualityCuts = [0.2,0.3,0.4]
+)
+
 from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
 from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-highPtTripletStep = TrackLwtnnClassifier.clone(
+trackdnn.toReplaceWith(highPtTripletStep, TrackLwtnnClassifier.clone(
     src = 'highPtTripletStepTracks',
     qualityCuts = [0.75, 0.775, 0.8],
-)
-fastSim.toModify(highPtTripletStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
-
-from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
+))
 
 highBetaStar_2018.toReplaceWith(highPtTripletStep, TrackMVAClassifierPrompt.clone(
      qualityCuts = [-0.2,0.3,0.4],
```

Comment:
this and pp_on_AA were previously simpler using toModify

Reply:
Changed back to toModify